src/nanoemoji/glue_together.py

```diff
@@ -34,7 +37,15 @@
 flags.DEFINE_string("target_font", None, "Font assets are copied into.")
 flags.DEFINE_string("donor_font", None, "Font from which assets are copied.")
 flags.DEFINE_string("color_table", None, "The color table to copy.")
-flags.DEFINE_string("output_file", None, "Font assets are copied into.")
+flags.mark_flag_as_required("target_font")
```

Comment:
why did you move these outside the `if __name__ == "__main__":` branch? I think that will make it raise as soon as one tries to import glue_together module. I think as long as these flags are only used inside main() function, it makes sense to only require them `if __name__ == "__main__"`

Reply:
Tbh I don't remember. I'll move them back until a reason comes to mind lol.